Adding a note for the eventrouter image (#70385)
openshift/openshift-docs: modules/cluster-logging-eventrouter-deploy.adoc

Use the following steps to deploy the Event Router into your cluster. You should always deploy the Event Router to the `openshift-logging` project to ensure it collects events from across the cluster.

[NOTE]
====

> The Event Router image is not a part of the Cluster Logging Operator and it needs to be downloaded separately.

Reviewer: ```suggestion
The Event Router image is not a part of the Cluster Logging Operator and needs to be downloaded separately.
```

> The Event Router image is not a part of the Cluster Logging Operator and it needs to be downloaded separately.

Reviewer 2: ```suggestion
The Event Router image is not a part of the {clo} and must be downloaded separately.
```
Please use the attribute for the Operator here.
